stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/usecase/EditMessageImplUseCaseTest.kt

```diff
@@ -30,7 +30,7 @@ internal class EditMessageImplUseCaseTest : BaseConnectedIntegrationTest() {
         messages = channelControllerImpl.messages.getOrAwaitValue()
         val liveLastMessage = messages.last()
         Truth.assertThat(liveLastMessage.id).isEqualTo(originalMessage.id)
-        Truth.assertThat(liveLastMessage.extraData).isEqualTo(updatedMessage.extraData)
+        Truth.assertThat(liveLastMessage.extraData).containsAtLeastEntriesIn(updatedMessage.extraData)
```

Comment:
We need to update those kinds of tests.
We can't depend on server implementation either network connections